packages/cli/cli/src/cli.ts

```diff
@@ -720,20 +720,32 @@ function addWriteOverridesCommand(cli: Argv<GlobalCliOptions>, cliContext: CliCo
     cli.command(
         "write-overrides",
         "Generate a basic openapi overrides file.",
-        (yargs) =>
+        (yargs) => [
             yargs.option("api", {
                 string: true,
                 description: "Only run the command on the provided API"
             }),
+            yargs.option("include-models", {
+                boolean: true,
+                description:
+                    "When generating the initial overrides, also stub the models (in addition to the endpoints)"
+            }),
+            yargs.option("existing-overrides", {
```

Comment:
can we detect this by looking at their `generators.yml` ? You should be able to get access to that from the TaskContext